packages/material-ui/src/Autocomplete/Autocomplete.test.js

```diff
@@ -1342,6 +1342,19 @@ describe('<Autocomplete />', () => {
         );
       }).toErrorDev('The Autocomplete expects the `value` prop to be an array or undefined.');
     });
+
+    it('warn if the type of the defaultValue is wrong', () => {
+      expect(() => {
+        PropTypes.checkPropTypes(
+          Autocomplete.propTypes,
+          { multiple: true, defaultValue: 'wrong-string', options: [], renderInput: () => null },
+          'prop',
+          'Autocomplete',
+        );
+      }).toErrorDev(
+        'The Autocomplete expects the `defaultValue` prop to be an array or undefined.',
```

Comment:
```suggestion
        'The Autocomplete expects the `defaultValue` prop to be an array when `multiple={true}` or undefined.',
```

Reply:
Should I change the warning for `value` too?

Reply:
too late now, should I open a different PR?

Reply:
A follow-up PR sounds great